templates/application-nginx-public-oauth2.yaml

```diff
@@ -0,0 +1,132 @@
+                                   "time_iso8601": "$time_iso8601", 
+                                   "msec": "$msec",
+                                   "remote_addr": "$remote_addr", 
+                                   "request": "$request",
+                                   "proxy_protocol_addr": "$proxy_protocol_addr", 
+                                   "x_forwarded_for": "$proxy_add_x_forwarded_for", 
+                                   "req_id": "$req_id",
+                                   "remote_user": "$remote_user",
+                                   "bytes_sent": $bytes_sent,
+                                   "request_time": $request_time,
+                                   "status": $status, 
+                                   "body_bytes_sent": "$body_bytes_sent",
+                                   "host": "$host",
+                                   "server_protocol": "$server_protocol",
+                                   "uri": "$uri",
+                                   "args": "$args",
+                                   "request_length": $request_length,
+                                   "proxy_upstream_name": "$proxy_upstream_name", 
+                                   "proxy_alternative_upstream_name": "$proxy_alternative_upstream_name",
+                                   "upstream_addr": "$upstream_addr",
+                                   "upstream_response_length": "$upstream_response_length",
+                                   "upstream_response_time": "$upstream_response_time", 
+                                   "upstream_status":"$upstream_status",
+                                   "request_method": "$request_method",
+                                   "http_referer": "$http_referer",
+                                   "namespace": "$namespace",
+                                   "ingress_name": "$ingress_name", 
+                                   "service_name": "$service_name", 
+                                   "service_port": "$service_port",
+                                   "http_host": "$http_host",
+                                   "scheme": "$scheme",
+                                   "server_name": "$server_name",
+                                   "upstream_cache_status": "$upstream_cache_status",
+                                   "request_id": "$request_id",
+                                   "ssl_protocol": "$ssl_protocol",
+                                   "ssl_cipher": "$ssl_cipher",
+                                   "http_user_agent": "$http_user_agent"
+                                   }' 
+          extraArgs:
+            default-ssl-certificate: glueops-core-cert-manager/{{ .Values.certManager.name_of_default_certificate }}
+          service:
+            annotations:
+              service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+              external-dns.alpha.kubernetes.io/hostname: ingress.{{ .Values.captain_domain }}
```

Comment:
can't use `ingress.*` as it'll step on the existing production/public one used for customer apps. Need to use something different

Reply:
I don't think we had one with pomerium. Maybe call this one authenticated-ingress. or glueops-ingress? Right now the use case is just glueops apps/control plane. What do you think would be a good name?